cartography/data/jobs/cleanup/aws_import_ec2_key_pairs_cleanup.json

```diff
@@ -1,20 +0,0 @@
-{
-  "statements": [
```

Comment:
This is covered by autocleanups in models.aws.ec2.keypairs

Reply:
Really awesome that we can do this partial refactor